frontend-react/src/components/SenderModeBanner.tsx

```diff
@@ -1,19 +1,19 @@
 import { ReactElement } from "react";
 import { IconWarning } from "@trussworks/react-uswds";
-import { NavLink } from "react-router-dom";
 
 import { useSessionContext } from "../contexts/SessionContext";
 import { MemberType } from "../hooks/UseOktaMemberships";
 import { useSenderResource } from "../hooks/UseSenderResource";
 
+import { USLink } from "./USLink";
+
 const isNotActive = (val: string | undefined): boolean => {
     return val === "testing" || val === "inactive";
 };
 
 const BannerContent = () => {
     const { senderDetail: sender, senderIsLoading: loading } =
         useSenderResource();
-    const path = "/getting-started/testing-facilities/overview";
```

Comment:
Seems like an unnecessary variable?  Glad we're simplifying it! 😆

Reply:
I mean, hey...sometimes you just gotta get those ugly strings in line 😂 but yeah, felt redundant.